components/grid/compact-grid.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Grid Size - Compact Grid
+page_title: Grid Size | Grid for Blazor
+description: Adjust the size and modify the appearance of the Telerik UI for Blazor Grid.
+slug: compact-grid
+tags: telerik,blazor,grid,size,compact
+published: true
+position: 20
+---
+
+## Grid Size
+
+This feature attempts to address the need for a **Compact Grid**, which renders more items by utilizing the available space, mainly through setting smaller padding in its cells.
+
+To utilize this feature, set the [`Size`](#grid-size) property of the component. It accepts values of type [`ThemeConstants.Grid.Size`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.ThemeConstants.Grid.Size) and the supported values are: 
+
+1. `Small` - Sets the padding of the building blocks of the Grid to **2px 8px**.
+1. `Medium` (Default) - Sets the padding of the building blocks of the Grid to **4px 8px**.
+
+Changing the `Size` property affects different building blocks of the component (tables, buttons, inputs, dropdowns, and others). 
+
+> The `Size` option does not affect elements displayed inside a popup (such as [Filter Menu](https://docs.telerik.com/blazor-ui/components/grid/filter/filter-menu), [Column Menu](https://docs.telerik.com/blazor-ui/components/grid/columns/menu)). 
+
+
+The following example demonstrates how to define the Grid size.
+
+```CSHTML
+<TelerikGrid Size="@ThemeConstants.Grid.Size.Small"
```

Comment:
You may set some height or paging here. The idea is to show a small Grid and with the current configuration, the component will expand on the whole page.